react/Viewer/controls.styl

```diff
@@ -76,6 +76,8 @@ $footerHeight = 3.5rem
 
     +medium-screen()
         height $toolbarHeightMedium
+        width calc(100% - .25rem)
+        padding 0 0 0 .25rem
```

Comment:
Why is the padding not symmetrical between left and right ? It seems like the reduced width is used for padding. Could it be "padding: 0 0.25rem" ?

Reply:
En fait pour avoir un comportement comme celui de la cozy-bar on a plutôt une marge à droite, rien à gauche, et un léger décalage sur le bouton

Reply:
OK, je trouve ca bizarre que ce soit pas symétrique. LGTM